docs/notebooks/ordinal_regression.ipynb

```diff
@@ -0,0 +1,1798 @@
+{
```

Comment:
<p>I think Pr(Y = 3) should be P(Z_1 &gt; tau_1) * P(Z_2 &gt; tau_2) * P(Z_3 &lt;= tau_3) which is (1 - P(Z_1 &lt;= tau_1)) * (1 - P(Z_2 &lt;= tau_2)) * P(Z_3 &lt;= tau_3)</p><p> </p><p>and then if the latent variable is normal, it would be (1 - Phi(tau_1)) * (1 - Phi(tau_2)) * Phi(tau_3), which is consistent with what we have here https://github.com/bambinos/bambi/blob/169564fe640818525049d7d946636266e4a7b968/bambi/families/univariate.py#L390</p><p> </p><p>and what you wrote below </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>Thanks for the clarification on the equations. That makes sense!</p>